downstream/assemblies/platform/assembly-install-aap-operator.adoc

```diff
@@ -11,6 +11,10 @@ ifdef::context[:parent-context: {context}]
 
 .Prerequisites
 * You have installed the {PlatformName} catalog in Operator Hub.
+* You have created a Storage Class if one does not exist. See link:https://docs.openshift.com/container-platform/4.10/storage/dynamic-provisioning.html[Dyamic Provisioning] for details.
+* To run {OCP} clusters on Amazon Web Services with ReadWriteMany, you must add NFS or other storage. 
```

Comment:
Where does NFS come from? This reads a little strange to me because RWX access mode is specific to how pods on the OCP cluster interacts with the persistent volume. The existence of RWX access mode implies there is a persistent volume. So it's like the wording is the other way round.

Also this sounds like a pre-req for OCP itself rather than AAP. Maybe it could be something more like:

You have added a persistent volume with the `ReadWriteMany` access mode.

Reply:
@oraNod The NFS information was previously part of a note placed just before the procedures. In the issue, it was stated that this is actually a pre-req and requested that all pre-reqs be included. 

These prerequisites are in the section for OCP so I think it's ok for it to come across that way, but I will update with your suggested change.

Reply:
Thanks @dcdacosta I hope I'm not adding noise with my review comments. I just think like it could be clarified a little bit more. I find looking at the procedures in the OCP docs helped me understand a little more: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/storage/index#storage-create-storage-class_persistent-storage-aws 

The first procedure is about creating a `StorageClass` then creating a PVC. So maybe we could make it a single pre-req like:

You have created the appropriate `StorageClass` object for your platform and a persistent volume claim (PVC) with `ReadWriteMany` access mode.

Anyway I'll leave you to it but do hope this is helpful in some way. Cheers.